src/components/SelectedMovie/SelectedMovie.css

```diff
@@ -72,11 +72,12 @@
 
 .sidebar {
     font-size: .75em;
-    border: 5px dotted yellow;
+    /* border: 5px dotted yellow; */
     margin-left: 2em;
     width: 12vw;
     height: 52vh;
```

Comment:
Great way to organize the information without adding extra containers onto the page.